webapp/backend/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+allowed.origins=http://splinter_frontend:4200
+server.port=8080
+spring.data.rest.base-path=/api
+
+spring.main.banner-mode=off
+spring.jpa.properties.hibernate.dialect= org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.show-sql=true
+
+spring.datasource.url=jdbc:postgresql://splinter_db:5432/splinter
+spring.datasource.username=postgres
+spring.datasource.password=1234
```

Comment:
Same as before, is there any more secure way to store those passwords?

Reply:
same as in https://github.com/cvlab-ai/splinter/pull/32#discussion_r979432357